src/utils/AlertsUtils.js

```diff
@@ -2,11 +2,14 @@
 import { PYTHON_APP } from './EnvUtils';
 import RequestUtils from './RequestUtils';
 
+const THREE_SEC_IN_MS = 3000;
```

Comment:
rename to something about the function of the constant rather than what it physically is (`FETCH_TIMEOUT` or something similar)

Reply:
i feel like this is fine because in the code it's: `timeout: THREE_SEC_IN_MS`, but @meganle can you decide here as well

Reply:
i think the name is fine as is